initCobraToolbox.m

```diff
@@ -171,7 +171,7 @@ function initCobraToolbox()
 %If the Matlab version is prior to Matlab 2016a, we have to use the TC.xml
 %structure to provide autocompletion, otherwise we can use the json
 %definition.
-if MatlabYear < 2015
+if MatlabYear < 2016

```

Comment:
On `R2014b` and `R2015b`, it just doesn't do anything (as if `tabcomplete` wasn't present).

Also, on `macOS`, same error:
````
Error parsing JSON data; Boost reports "<unspecified file>(11): expected ',' or '}'"
````